docs/user-guide/replicasets/hpa-rs.yaml

```diff
@@ -3,11 +3,10 @@ kind: HorizontalPodAutoscaler
 metadata:
   name: frontend-scaler
 spec:
-  scaleRef:
+  scaleTargetRef:
     kind: ReplicaSet
     name: frontend
-    subresource: scale
   minReplicas: 3
   maxReplicas: 10
   cpuUtilization:
```

Comment:
The struct is no flattened. `cpuUtilization` should be removed and `targetCPUUtilizationPercentage` should be top level field in spec.

See http://kubernetes.io/docs/api-reference/autoscaling/v1/definitions/#_v1_horizontalpodautoscalerspec for more details